render/process.go

```diff
@@ -87,18 +87,27 @@ var ProcessNameRenderer = ConditionalRenderer(renderProcesses,
 
 // MapEndpoint2Pseudo makes internet of host pesudo nodes from a endpoint node.
 func MapEndpoint2Pseudo(n report.Node, local report.Networks) report.Nodes {
-	_, addr, _, ok := report.ParseEndpointNodeID(n.ID)
+	id, ok := pseudoNodeID(n, local)
 	if !ok {
 		return report.Nodes{}
 	}
+	externalNode := NewDerivedPseudoNode(id, n)
+	return report.Nodes{externalNode.ID: externalNode}
+}
```

Comment:
Inverting the condition would allow you to inline the first line.

Reply:
function NewDerivedExternalNode has been removed now